### ISSUE TITLE
Release SQL-ODBC drivers

### ISSUE THREAD

**Issue by joshuali925** [opensearch-project/opensearch-build#1872]
### Is your feature request related to a problem? Please describe

SQL ODBC had a few bug fixes since previous release, we would like to initiate a new release for ODBC drivers to unblock some users. The artifacts are listed below:

Driver | Platform | Type
-- | -- | --
ODBC | Mac | zip
ODBC | Windows32 | msi
ODBC | Windows64 | msi

### Describe the solution you'd like

Help from infra in signing and releasing the artifacts

**Comment by gaiksaya** [comment]
Hi @joshuali925 , can you start looking into automating this? From our offline conversation, I think the testing step can be done before forwarding the artifacts to our team for release. 
I am not sure how these artifacts are generated today but the automated workflow would look like 
* Download the artifacts from given location (eg: GHA workflows artifacts, s3, etc)
* Sign them
* Upload them to production S3 bucket. 

All of the above steps can be put into a JekinsFile. Example of one such component is data prepper: https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile

Thanks!

**Comment by gaiksaya** [comment]
Yes, the signing will be same depending on the type of artifact. Looks like it is zip (maybe linux? not sure what we used before for zips) and windows. 

Yes uploading to s3 will be same:
https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile#L68-L76
The path will be same what we used before in manual process.

Regarding testing I believe since this is directly affecting the production workflow, testing would be difficult. The underlying libraries are well tested. However, you can add tests for your jenkinsFile that shows the regression basically what calls are made. More details here: https://github.com/opensearch-project/opensearch-build/blob/main/DEVELOPER_GUIDE.md#jenkins-pipelines-and-shared-libraries
If you still want to play around with jenkins, please see https://github.com/opensearch-project/opensearch-ci which will allow you to set up your own instance by executing few commands.

Thanks!
cc: @bbarani

**Comment by zelinh** [comment]
@mengweieric Can we close this issue here since we already released the ODBC?

**Comment by dharminfadia** [comment]
> @mengweieric Can we close this issue here since we already released the ODBC?
Hello Guys I am having same issue can you brief step how can solve this ..

**Comment by dharminfadia** [comment]
> @dharminfadia the new ODBC drivers are released on the bottom of https://opensearch.org/artifacts, could you try if they would work? If not feel free to create an issue under https://github.com/opensearch-project/sql/issues/
> 
> Closing this release issue as it's already done





Hello @joshuali925
![image](https://user-images.githubusercontent.com/87129707/169454143-d6c86b3f-bd7d-4695-a3e0-af89c1895c6e.png)
latest Driver is install still having same issue
![image](https://user-images.githubusercontent.com/87129707/169454220-a6fa0ea0-a72b-45a3-9a66-82347c8c2f48.png)
I am using Opensearch version is 1.2.4 but still facing issue .
